content/en/community.md

```diff
@@ -15,8 +15,4 @@ Docs and other calendar invites may be shared directly with this group, so pleas
 
 ## Slack
 
-You can also keep in touch by joining our [Slack channel](https://sigstore.slack.com). Use [this invite link](https://links.sigstore.dev/slack-invite) to join.
-
-[//]: # (In case the invite link is expired, ping Dan on Slack of via Twitter: @lorenc_dan)
-
-_Having issues to join Slack? Ping [@lorenc_dan](https://twitter.com/lorenc_dan) on Twitter._
+You can also keep in touch by joining our [Slack channel](https://sigstore.slack.com). Use [this invite link](https://links.sigstore.dev/slack-invite) to join.
```

Comment:
I thought we could maybe leave a single person out of this. If we need a troubleshooting option maybe it can be to tag https://twitter.com/projectsigstore

Reply:
Yeah good point. Dan asked to add some comments close to where the invite link is just in case it expires again, and he said to point to him, but I was unsure if it should be a hidden markdown comment or a note. I think adding the note pointing people to the Twitter account is a good option to still have some troubleshooting option visible in the docs. I can update it